### ISSUE TITLE
Open source CPFT code?

### ISSUE THREAD

**Issue by hdzhang-code** [jianguoz/Few-Shot-Intent-Detection#4]
Hi, Jianguo,

Congratulations to you as the Research Scientist in Saleforces.

It is highly appreciated if you can open source the CPFT code. I fail to reproduce the results in the paper. Is there any critical training details, such as learning rate scheduler?

Best Regards,

**Comment by ZIZUN** [comment]
@hdzhang-code 
I implemented CPFT unofficial version codes.
if you need, check this repo (https://github.com/ZIZUN/CPFT).
